lib/attach-middleware.js

```diff
@@ -25,22 +24,12 @@ function installRouter(app) {
  * Creates a new coverage map on every request.
  *
  * @param {express.Application} app
+ * @param {any} _options
  */
-const serverMiddleware = (app, _options) => {
-  installRouter(app);
-};
-
-/**
- * Used when app is in ci mode (`ember test`).
- * Collects the coverage on each request and merges it into the coverage map.
- *
- * @param {express.Application} app
- */
-const testMiddleware = (app, _options) => {
+const middleware = (app, _options) => {
```

Comment:
Its not immediately obvious why we need this function at all?

Wouldn't it be the same to replace callers with `installRouter`?

What am I missing?

Reply:
Fixed 2934108